pkgs/applications/version-management/git-vanity-hash/default.nix

```diff
@@ -24,5 +24,10 @@ rustPlatform.buildRustPackage rec {
     license = [ licenses.mit ];
     maintainers = [ maintainers.kaction ];
     mainProgram = "git-vanity-hash";
+    # Broken due to ancient dependency rusc-serialize
+    # https://hydra.nixos.org/build/256865980/nixlog/2
+    # Potential fix submitted upstream
+    # https://github.com/prasmussen/git-vanity-hash/pull/2
```

Comment:
Why not fetch the patch instead?

Reply:
I'll give it a try!

Reply:
I added a commit to patch this, but it won't build complaining about wrong cargoHash. I updated it as usual, but it's still failing. I asked for some tips about the issue, and will gladly accept any help on this @SuperSandro2000! Thanks in advance!